api/lib/infrastructure/monitoring-tools.js

```diff
@@ -26,46 +55,40 @@ function logKnexQueriesWithCorrelationId(data, msg) {
   }
 }
 
-function logErrorWithCorrelationId(error) {
-  const request = asyncLocalStorage.getStore();
-  logger.error({
-    request_id: `${get(request, 'info.id', '-')}`,
-    http: {
-      method: get(request, 'method', '-'),
-      url_detail: {
-        path: get(request, 'path', '-'),
-      },
-    },
-  }, error);
-}
-
-function logInfoWithCorrelationId(message) {
-  const request = asyncLocalStorage.getStore();
-  logger.info({
-    request_id: `${get(request, 'info.id', '-')}`,
-    http: {
-      method: get(request, 'method', '-'),
-      url_detail: {
-        path: get(request, 'path', '-'),
-      },
-    },
-  }, message);
-}
-
-function addPositionToQuerieAndIncrementQueriesCounter(knexQueryId) {
+function addKnexMetricsToRequestContext(data) {
   const request = asyncLocalStorage.getStore();
   if (request) {
     request.knexQueryPosition = request.knexQueryPosition || [];
+    request.knexQueries = request.knexQueries || [];
     request.queriesCounter = request.queriesCounter || 0;
     request.queriesCounter++;
-    request.knexQueryPosition[knexQueryId] = request.queriesCounter;
+    request.knexQueryPosition[data.__knexQueryUid] = request.queriesCounter;
+    request.knexQueries.push({
+      knex_query_id: data.__knexQueryUid,
+      knex_query_position: request.queriesCounter,
+      knex_query_sql: data.sql,
+      knex_query_params: [(data.bindings) ? data.bindings.join(',') : ''],
+      duration: get(data, 'duration', '-'),
+    });
   }
 }
 
+function extractUserIdFromRequest(request) {
+  let userId = get(request, 'auth.credentials.userId');
+  if (!userId && get(request, 'headers.authorization')) userId = requestUtils.extractUserIdFromRequest(request);
+  return userId || '-';
+}
+
+function isStoreContainsApiRequest() {
```

Comment:
`is`…`Contains` ça fait deux verbes qui ne vont pas vraiment ensemble. Suggestions : `doesStoreContainApiRequest` ou encore `storeContainsApiRequest`. Le préfixe `Api` n'est pas très utile d'ailleurs je trouve, on pourrait s'en débarrasser.